virttest/utils_tmux.py

```diff
@@ -0,0 +1,26 @@
+"""
+The tmux command related utility functions
+"""
+
+import logging
+
+from virttest import utils_misc
+from virttest import utils_package
+
+
+def run_tmux_cmd(cmd, session=None, ignore_status=False):
+    """
+    Execute shell-commands in tmux on localhost or remote
+
+    :param cmd: The command line to run
+    :param session: The session object to the host
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: command status and output
+    """
+    if not utils_package.package_install("tmux", session):
+        logging.error("Failed to install required package - tmux!")
+    tmux_cmd = 'tmux -c "{}"'.format(cmd)
```

Comment:
The target is to add 'tmux -c' before a command. So I do not think we need to add a new function. What we need to do is only to combine 'tmux -c' with your target command together.

Reply:
Generally, I indent to close this PR.

Reply:
Ok, since it's a simple function, I will do it in my tp cases.